README.md

```diff
@@ -155,7 +155,7 @@ At a high level, we want ML researchers to:
 
 # Testing
 
-We use circleci to test on PyTorch versions 1.5.1, 1.6.0 and 1.7.0 and CUDA version 10.1. Please create an [issue](https://github.com/facebookresearch/fairscale/issues) if you are having trouble with installation.
+We use circleci to test on PyTorch versions 1.5.1, 1.6.0 and 1.7.1 and CUDA version 10.1. Please create an [issue](https://github.com/facebookresearch/fairscale/issues) if you are having trouble with installation.
```

Comment:
I use cuda 11 at home. Do circleCI servers support it?

Reply:
it doesn't matter what circleCI has - pytorch comes with its own cudatoolkit, see: https://pytorch.org/get-started/locally/

fairscale at the moment chooses to test w/ cuda-10.1, but they could test with cuda-11.0 (and 11.2 soon), as pt-1.7.1 w/ cuda-11.0 is available too.

Reply:
+1 on testing with cuda 11 since that seems to the default pip install torch version (#267). Is there a reason we want to only test with cuda 10.1?

Reply:
I am guess that we test 10.1 because that's the default of FAIR cluster.

Reply:
We are limited by the driver version:

nvidia-smi
Fri Jan 15 23:51:12 2021       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 418.87.00    Driver Version: 418.87.00    CUDA Version: 10.1     |

This Nvidia Linux kernel driver installed on the circleci image we are using only support up to Cuda 10.1:

https://docs.nvidia.com/deploy/cuda-compatibility/index.html

We can install Cuda 11.0 but it is incompatible with the driver the Linux kernel is using.

We may be able to work around this by building our own docker image.

Reply:
resolving this since it is not needed for this PR.

Reply:
Agreed. I think we may need to use a different image in the future for that. Definitely not meant to request this change here. I was ignorant about what is needed when I commented on this.

![image](https://user-images.githubusercontent.com/24926999/104790535-4fa0c780-574c-11eb-859c-afefe02eef23.png)